From 087a1ec0e60eec839e0e7be0cc8a23423dbce579 Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Sun, 27 Aug 2017 18:10:38 +0200
Subject: [PATCH 46/97] Explicitly call KAboutData::setDesktopFileName

Summary: Without this I don't get an application icon on Wayland.

Test Plan: Icon appears on Wayland

Reviewers: #choqok, scarpino

Differential Revision: https://phabricator.kde.org/D7576
---
 choqok/main.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/choqok/main.cpp b/choqok/main.cpp
index 5c8cb9fb..34b1a0bb 100644
--- a/choqok/main.cpp
+++ b/choqok/main.cpp
@@ -47,6 +47,7 @@ int main(int argc, char **argv)
                      QString(), QLatin1String("http://choqok.gnufolks.org/"));
 
     about.setOrganizationDomain("kde.org");
+    about.setDesktopFileName(QStringLiteral("org.kde.choqok"));
 
     about.addAuthor(i18n("Mehrdad Momeny"), i18n("Author, Developer and Maintainer"),
                     QLatin1String("mehrdad.momeny@gmail.com"), QLatin1String("http://momeny.wordpress.com"));
-- 
2.17.1

